src/pages/Auth/Auth.css

```diff
@@ -0,0 +1,29 @@
+
+ .bg-img{
+    width: 50%; 
+    height:  90%; 
+    object-fit: cover;
+}
+
+.input{
+    width: 22rem;
+}
+
+.divider{
+    color: #A14F57;
```

Comment:
Suggestion: you can use variable color naming here

Reply:
changes done!